pfcpiface/telemetry.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+// PfcpStats ... Prometheus metrics
```

Comment:
Does this struct need to exported? Can we instead start it with small letter instead?
```
type pfcpStats struct {
```